tests/test_torchinductor.py

```diff
@@ -2033,6 +2035,20 @@ def fn(a):
 
         self.common(fn, [torch.randn(55)])
 
+    @patch.object(torchinductor.config.triton, "cudagraphs", True)
+    def test_dropout(self):
+        random.seed(1234)
```

Comment:
would be good to test that a function with the same seed produces deterministic results, and that replaying cuda graph for a second time (without setting seed) produces different results

Reply:
I added a test for this